README.md

```diff
@@ -29,3 +29,8 @@ You can easily install them via the conda command:
 ```
 conda install -c scitools  -c https://conda.anaconda.org/opengeostat numpy scipy matplotlib ipython notebook netCDF4 progressbar vtk cartopy xarray dask bottleneck pyevtk numexpr basemap
 ```
+
+## Running the analysis
+  1. create a configuration file by copying and modifying `config.template` or one of the example files in `configs`
```

Comment:
Should we exclude the custom config files in gitignore? A quick google search revealed that we could have something like this in .gitignore:
`config.*
!config.template`

which basically means ignore all config.\* *but*  config.template.